include/mata/nfa/algorithms.hh

```diff
@@ -68,6 +68,8 @@ Nfa complement_classical(const Nfa& aut, const Mata::Util::OrdVector<Symbol>& sy
  * i.e., if the final intersection of smaller complement of bigger is empty.
  */
 bool is_included_naive(const Nfa& smaller, const Nfa& bigger, const Alphabet* alphabet = nullptr, Run* cex = nullptr);
+//naive is not a good word, maybe textbook, or explicit, construct_first
+//the alphabet parameter is useless in inclusion
```

Comment:
Discussions are private, code is public. I would go for `textbook`.

Reply:
If the algorithm, however basic, has a name, use the name instead?

Reply:
I like the `explicit`. I have read a textbook with the antichain-based inclusion checking. Basically, you could append `textbook` to every function name in mata.

Reply:
I don't like naive because it is not really clear what it is outside our group and it is somehow pejorative. 
I would like textbook, I think everybody will know, I don't think it can be really confused with antichains. Explicit is ok too, but it requires a lot of context to get the meaning.